### PR TITLE
Go back to checked-in minified .js files

### PR DESCRIPTION
**Pull request by wtgodbe**
Part of https://github.com/dotnet/aspnetcore/issues/56933. Hoping to backport to preview7. Still need to create a github action for updating the .js files as a convenience for devs

### CODE REVIEW

**Comment by wtgodbe** [comment]
@javiercn hitting errors w/ https://github.com/dotnet/aspnetcore/blob/main/eng/scripts/npm/update-dependency-versions.mjs, would you mind taking a look?

**Comment by JamesNK** [comment]
Is this a fix for https://dev.azure.com/dnceng/internal/_build/results?buildId=2500435&view=logs&j=316d5c15-0c50-544e-8051-e6b14a1ab674&t=4e01507e-10cc-507d-f6d0-adc59e92bcb8? (from build ops active tasks)

**Comment by wtgodbe** [comment]
> Is this a fix for https://dev.azure.com/dnceng/internal/_build/results?buildId=2500435&view=logs&j=316d5c15-0c50-544e-8051-e6b14a1ab674&t=4e01507e-10cc-507d-f6d0-adc59e92bcb8? (from build ops active tasks)

That was fixed by https://github.com/dotnet/aspnetcore/pull/56954 - this PR goes further & basically replaces the original one that introduced that build break

File: src/Components/Web.JS/.gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 ﻿node_modules/
-dist/
-dist/
+dist/Debug/
+dist/Release/blazor.webassembly.js
+dist/Release/blazor.webview.js
```
Comment (javiercn) [inline src/Components/Web.JS/.gitignore:4]: If this is ignored, how do these files get updated by devs when they make changes.

File: src/Components/Web.JS/.gitignore [hunk 2]
```diff
@@ -1,3 +1,4 @@
 ﻿node_modules/
-dist/
-dist/
+dist/Debug/
```
Comment (javiercn) [inline src/Components/Web.JS/.gitignore:2]: Ignoring debug is fine

File: eng/scripts/update-npm-dependencies.ps1 [hunk 3]
```diff
@@ -1,132 +0,0 @@
...
-if ($WhatIf -or $SkipPullRequestCreation) {
-    Write-Host "Skipping pull request creation for the root of the repository"
-}
-else {
-    Write-Host "Creating pull request for the root of the repository"
-    git branch --set-upstream-to=origin/main
-    git push origin $branchName`:$branchName --force;
-    gh repo set-default dotnet/aspnetcore
-    gh pr create --base main --head $branchName --title "[Infrastructure] Updated npm package cache $(Get-Date -Format 'yyyy-MM-dd')" --body ""
-}
```
Comment (javiercn) [inline eng/scripts/update-npm-dependencies.ps1:132]: The script is still useful, just not the submodules part.  Leave it around

**Review by javiercn — APPROVED** [review]

File: eng/scripts/update-npm-dependencies.ps1 (hunk 3 above)
Comment (wtgodbe) [reply eng/scripts/update-npm-dependencies.ps1]: Done

**Pull request merged**